lib/rufo/dot_file.rb

```diff
@@ -6,9 +6,7 @@ def initialize
   end
 
   def find_in(dir)
-    @cache.fetch(dir) do
-      @cache[dir] = internal_find_in(dir)
-    end
+    @cache[dir] ||= internal_find_in(dir)
```

Comment:
@splattael it's more elegant way to memoize dirs. And it memoizes `nil` values too:

```ruby
@cache = {}

def find_in(dir)
  # new method body
end

def internal_find_in(dir)
  # method body
end

find_in('/home/users/projects/project_name')
@cache #=> {"/"=>nil, "/home"=>nil, "/home/users"=>nil, "/home/users/projects"=>nil, "/home/users/projects/project_name"=>nil}
```